### PR TITLE
Update push-api.json

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

**Comment by jpmedley** [comment]
@Fyrd I think there might be a bug in your Travis script. It doesn't like either of these items in my PR. I can't find anything wrong with either of them. I'f you see something I don't, I'll be happy to fix it.

"and_chr":{
      "49":"a #1",
      "50":"y"
      },

"and_chr":{
      "50":"y"
      },

File: features-json/push-api.json [hunk 1]
```diff
@@ -227,7 +227,7 @@
       "35":"n"
     },
     "and_chr":{
-      "49":"a #1"
+      "50":"y"
```
Comment (cvrebert) [inline features-json/push-api.json]: This is the error. Only 1 version of Android Chrome is currently allowed.

Comment (jpmedley) [inline features-json/push-api.json]: I updated the commit so that it only has one version. The CI check is still failing.

Comment (cvrebert) [inline features-json/push-api.json]: Erm, rather, only data for what caniuse considers the "current" Android Chrome version is allowed. You'd need @Fyrd to bump that version from 49 to 50.

Comment (Fyrd) [inline features-json/push-api.json]: Right, can only update this once Chrome 50 for Android is out.

**Comment by Fyrd** [comment]
Updated this one manually.

**Pull request closed without merging**